src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -20,15 +28,53 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
+        String trimmedArgs = " 0 " + args.trim();
```

Comment:
Would you consider declaring a final constant to give meaning to `" 0 "`?

Reply:
good idea!